lua/nvim-tree/git.lua

```diff
@@ -87,9 +115,18 @@ function M.update_status(entries, cwd)
     return
   end
 
+  if not parent_node then parent_node = {} end
+
   local matching_cwd = utils.path_to_matching_str( utils.path_add_trailing(git_root) )
+  local num_ignored = 0
 
   for _, node in pairs(entries) do
+    if parent_node.git_status == "ignored" or M.should_gitignore(node.absolute_path) then
+      node.git_status = "ignored"
+      num_ignored = num_ignored + 1
+      goto continue
```

Comment:
i believe we should avoid goto because it might not match all users lua version (i remember having this issue before and i had to remove the gotos).

Reply:
Gotcha. I wasn't aware that it had limited support. Thanks.